ui/src/components/WorkerManagerWorkerTypesTable/index.jsx

```diff
@@ -0,0 +1,154 @@
+import { WorkerManagerWorkerTypeSummary } from '../../utils/prop-types';
+import DataTable from '../DataTable';
+import sort from '../../utils/sort';
+import Link from '../../utils/Link';
+import DateDistance from '../DateDistance';
+
+export default class WorkerManagerWorkerTypesTable extends Component {
+  static propTypes = {
+    workerTypes: arrayOf(WorkerManagerWorkerTypeSummary).isRequired,
+    searchTerm: string,
+    path: string,
+  };
+
+  state = {
+    sortBy: null,
+    sortDirection: null,
+  };
+
+  sortWorkerTypes = memoize(
+    (workerTypes, sortBy, sortDirection, searchTerm) => {
+      const sortByProperty = camelCase(sortBy);
+      const filteredWorkerTypes = searchTerm
+        ? workerTypes.filter(({ workerType }) =>
+            workerType.includes(searchTerm)
+          )
+        : workerTypes;
+
+      return isEmpty(filteredWorkerTypes)
+        ? filteredWorkerTypes
+        : [...filteredWorkerTypes].sort((a, b) => {
+            const firstElement =
+              sortDirection === 'desc' ? b[sortByProperty] : a[sortByProperty];
+            const secondElement =
+              sortDirection === 'desc' ? a[sortByProperty] : b[sortByProperty];
+
+            return sort(firstElement, secondElement);
+          });
+    },
+    {
+      serializer: ([workerTypes, sortBy, sortDirection, searchTerm]) => {
+        const ids = pipe(
+          rSort((a, b) => sort(a.workerType, b.workerType)),
+          map(({ workerType }) => workerType)
+        )(workerTypes);
+
+        return `${ids.join('-')}-${sortBy}-${sortDirection}-${searchTerm}`;
+      },
+    }
+  );
+
+  handleHeaderClick = sortBy => {
+    const toggled = this.state.sortDirection === 'desc' ? 'asc' : 'desc';
+    const sortDirection = this.state.sortBy === sortBy ? toggled : 'desc';
+
+    this.setState({ sortBy, sortDirection });
+  };
+
+  renderRow = workerType => {
+    const { path } = this.props;
+    const iconSize = 16;
+    const { failedNumber, exceptionNumber, unscheduledNumber } = workerType;
+
+    return (
+      <TableRow key={workerType.name}>
+        <TableCell
+          button
+          component={Link}
+          to={`${path}/worker-types/${workerType.name}`}>
+          <Typography>{workerType.name}</Typography>
+          <LinkIcon size={iconSize} />
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.pendingCapacity}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.runningCapacity}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.pendingTasks}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>
+            <DateDistance from={workerType.lastActive} />
+          </Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>
+            <DateDistance from={workerType.lastResolved} />
+          </Typography>
+        </TableCell>
```

Comment:
Add an option to copy the date in its raw form similar to https://taskcluster-web.netlify.com/provisioners/aws-provisioner-v1.

![Screen Shot 2019-04-22 at 9 08 44 AM](https://user-images.githubusercontent.com/3766511/56501834-41256d80-64de-11e9-9528-e5eacbe8f8e8.png)